cmd/gateway-common.go

```diff
@@ -403,6 +403,10 @@ func gatewayHandleEnvVars() {
 		}
 		globalPanFSDefaultBucketPath = path
 
+		panOnlyLocalAdminApi := env.Get("MINIO_PANFS_ONLY_LOCAL_ADMIN_API", "1")
```

Comment:
If you change `panOnlyLocalAdminApi` to `panfs...`, grepping for PanFS-related changes will be easier in the future.